README.md

```diff
@@ -1,10 +1,10 @@
-# Atlan dbt Action
+# Atlan Action
 
 ![atlan<>dbt](https://user-images.githubusercontent.com/14099191/209542321-54d5557e-8abf-4d9a-9f6d-dcacb856f25f.png)
 
 ## Overview
 
-_Have you ever changed a dbt model only to later find it broke a downstream table or dashboard? 💔_
+_Have you ever changed a dbt model or a data contract only to later find it broke a downstream table or dashboard? 💔_
```

Comment:
@rittikdasgupta Let us add the hyperlink to the docs of data contracts in the word?